app/models/user.rb

```diff
@@ -1,10 +1,13 @@
 class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+  devise :database_authenticatable, :registerable,
```

Comment:
Kindly add the `:confirmable` module to ensure that a user confirms their email.

<img width="892" alt="image" src="https://github.com/BahirHakimy/Rails-Blog/assets/27709832/05d8cc5f-38d4-4924-bc94-15383312e47c">